OSBotty/src/defaultPackage/main.java

```diff
@@ -205,19 +205,19 @@ public void actionPerformed(ActionEvent e) {
 	public static void guiLooper(final JLabel counterLabel) {
         switch(loopCount) {

```

Comment:
We may want to change the output loop functionality later in order to make it more readable